server/odc-service/src/main/java/com/oceanbase/odc/metadb/dlm/TaskGeneratorRepository.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.metadb.dlm;
+
+import java.util.Optional;
+
+import com.oceanbase.odc.config.jpa.OdcJpaRepository;
+
+/**
+ * @Author：tinker
+ * @Date: 2023/8/10 14:48
+ * @Descripition:
+ */
+public interface TaskGeneratorRepository extends OdcJpaRepository<TaskGeneratorEntity, Long> {
+
+    Optional<TaskGeneratorEntity> findByJobId(String jobId);
+
+    Optional<TaskGeneratorEntity> findByGeneratorId(String generatorId);
```

Comment:
how can you guarantee that you will get a singleton object queried by generatorId? I don't see any constraints on it

Reply:
This method is unused and has been deleted.